### ISSUE TITLE
Guest-specific operations were attempted... vmware_workstation provider

### ISSUE THREAD

**Issue by scott-mead** [mitchellh/vagrant#7145]
- Host: Fedora 23
- Provider: vmware_workstation
- Vmware: Workstation 12 pro (12.1.0 build-3272444)
- Vagrant: Vagrant 1.8.1

```
$ vagrant plugin list
vagrant-share (1.1.5, system)
vagrant-vmware-workstation (4.0.8)
```

Box: Custom (metadata.json attached)
- box os: centos 6.7
- vagrant user with vagrant public key

Provisioner: chef-solo (motd cookbook for testing)

Attached: 
- [metadata.json.txt](https://github.com/mitchellh/vagrant/files/176098/metadata.json.txt)
- [Vagrantfile.txt](https://github.com/mitchellh/vagrant/files/176092/Vagrantfile.txt)
- [vagrantup.log.txt](https://github.com/mitchellh/vagrant/files/176100/vagrantup.log.txt)
- [vagrantdebugup.log.txt](https://github.com/mitchellh/vagrant/files/176102/vagrantdebugup.log.txt)

Description:
- If I don't use a versioned metadata.json, the vm starts fine
- If I run VAGRANT_LOG=debug vagrant up, the vm _sometimes_ starts fine
- I I run vagrant up, the following occurs:

```
$ vagrant up
Bringing machine 'default' up with 'vmware_workstation' provider...
==> default: Cloning VMware VM: 'openscg/centosbox'. This can take some time...
==> default: Checking if box 'openscg/centosbox' is up to date...
==> default: Verifying vmnet devices are healthy...
==> default: Auto-generating node name for Chef...
==> default: Preparing network adapters...
==> default: Starting the VMware VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 172.16.87.132:22
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Host unreachable. Retrying...
    default: Warning: Host unreachable. Retrying...
    default: 
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default: 
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Forwarding ports...
    default: -- 22 => 2222
==> default: Setting hostname...
Guest-specific operations were attempted on a machine that is not
ready for guest communication. This should not happen and a bug
should be reported.
```

**Comment by sethvargo** [comment]
Hi @scott-mead 

Thank you for opening an issue. Are you able to reproduce this with the hashicorp/precise64 box? This will help us in narrowing down if it's a box issue or Vagrant core issue. Thanks!

**Comment by scott-mead** [comment]
Using hashicorp/precise64 seems to work.  "Setting Hostname" takes a while, but, it works.  Any pointers on where to look in the private box for issues?

**Comment by sethvargo** [comment]
Hi @scott-mead 

There seem to be a number of issues with the centos boxes as of late - could you try using one of the boxes under the `bento` project?

**Comment by scott-mead** [comment]
Sorry, I can't use those.  Honestly, this looks like a problem with the VMware provider.  I notice a couple of things, the biggest is that my system tray starts notifying me that the vmnet network devices are changing state right when this happens.  Is the vmware kit for linux messing the state of the virtual interfaces?

==> default: Forwarding ports...
 INFO interface: detail: -- 22 => 2222
 INFO interface: detail:     default: -- 22 => 2222
    default: -- 22 => 2222
 INFO vmware_driver: Forwarding 1 ports.
 INFO subprocess: Starting process: ["/home/scottm/.vagrant.d/gems/gems/vagrant-vmware-workstation-4.0.8/bin/vagrant_vmware_desktop_sudo_helper_wrapper_linux_amd64", "forward-ports", "/tmp/vagrant-sudo-helper20160321-3428-1a7l2gd"]
 INFO warden: Calling OUT action: VMware Middleware: ForwardPorts
 INFO warden: Calling OUT action: #<Vagrant::Action::Builtin::HandleForwardedPortCollisions:0x000000024de4f8>
 INFO warden: Calling OUT action: VMware Middleware: PrepareForwardedPortCollisionParams
 INFO warden: Calling OUT action: VMware Middleware: VMXModify
 INFO interface: info: Setting hostname...
 INFO interface: info: ==> default: Setting hostname...
==> default: Setting hostname...
 INFO ssh: Connection errored, not re-using. Will reconnect.
 INFO subprocess: Starting process: ["/usr/bin/vmrun", "list"]
 INFO subprocess: Command not in installer, restoring original environment...
 INFO vmware_driver: Reading an accessible IP for machine...
 INFO vmware_driver: Trying vmrun getGuestIPAddress...
 INFO subprocess: Starting process: ["/usr/bin/vmrun", "getGuestIPAddress", "/home/scottm/work/git/scotts-scripts/vagrant/auth/.vagrant/machines/default/vmware_workstation/6f6d366b-a3b9-4226-8b8e-fd3c00381ce4/centosbox.vmx"]
 

> **INFO subprocess: Command not in installer, restoring original environment...**

 INFO vmware_driver: vmrun getGuestIPAddress success: 172.16.87.136
 INFO ssh: Attempting SSH connection...
 INFO ssh: Attempting to connect to SSH...
 INFO ssh:   - Host: 172.16.87.136
 INFO ssh:   - Port: 22
 INFO ssh:   - Username: vagrant
 INFO ssh:   - Password? false
 INFO ssh:   - Key Path: ["/home/scottm/work/git/scotts-scripts/vagrant/auth/.vagrant/machines/default/vmware_workstation/private_key"]

> ** INFO retryable: Retryable exception raised: #<Errno::EHOSTUNREACH: No route to host - connect(2) for 172.16.87.136:22>**

 INFO ssh: Attempting to connect to SSH...
 INFO ssh:   - Host: 172.16.87.136
 INFO ssh:   - Port: 22
 INFO ssh:   - Username: vagrant
 INFO ssh:   - Password? false
 INFO ssh:   - Key Path: ["/home/scottm/work/git/scotts-scripts/vagrant/auth/.vagrant/machines/default/vmware_workstation/private_key"]
 INFO retryable: Retryable exception raised: #<Errno::EHOSTUNREACH: No route to host - connect(2) for 172.16.87.136:22>
 INFO ssh: Attempting to connect to SSH...
 INFO ssh:   - Host: 172.16.87.136
 INFO ssh:   - Port: 22
 INFO ssh:   - Username: vagrant
 INFO ssh:   - Password? false
 INFO ssh:   - Key Path: ["/home/scottm/work/git/scotts-scripts/vagrant/auth/.vagrant/machines/default/vmware_workstation/private_key"]
 INFO retryable: Retryable exception raised: #<Errno::EHOSTUNREACH: No route to host - connect(2) for 172.16.87.136:22>
 INFO ssh: Attempting to connect to SSH...
 INFO ssh:   - Host: 172.16.87.136
 INFO ssh:   - Port: 22
 INFO ssh:   - Username: vagrant
 INFO ssh:   - Password? false
 INFO ssh:   - Key Path: ["/home/scottm/work/git/scotts-scripts/vagrant/auth/.vagrant/machines/default/vmware_workstation/private_key"]
 INFO retryable: Retryable exception raised: #<Errno::EHOSTUNREACH: No route to host - connect(2) for 172.16.87.136:22>
 INFO ssh: Attempting to connect to SSH...
 INFO ssh:   - Host: 172.16.87.136
 INFO ssh:   - Port: 22
 INFO ssh:   - Username: vagrant
 INFO ssh:   - Password? false
 INFO ssh:   - Key Path: ["/home/scottm/work/git/scotts-scripts/vagrant/auth/.vagrant/machines/default/vmware_workstation/private_key"]
 INFO ssh: SSH not up: #<Vagrant::Errors::SSHNoRoute: While attempting to connect with SSH, a "no route to host" (EHOSTUNREACH)
error was received. Please verify your network settings are correct
and try again.>
ERROR warden: Error occurred: Guest-specific operations were attempted on a machine that is not
ready for guest communication. This should not happen and a bug
should be reported.
 INFO warden: Beginning recovery process...
 INFO warden: Calling recover: #<Vagrant::Action::Builtin::HandleForwardedPortCollisions:0x000000024de4f8>
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
ERROR warden: Error occurred: Guest-specific operations were attempted on a machine that is not
ready for guest communication. This should not happen and a bug
should be reported.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
ERROR warden: Error occurred: Guest-specific operations were attempted on a machine that is not
ready for guest communication. This should not happen and a bug
should be reported.
 INFO warden: Beginning recovery process...
 INFO warden: Calling recover: #<Vagrant::Action::Builtin::Call:0x000000021b9db8>
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
ERROR warden: Error occurred: Guest-specific operations were attempted on a machine that is not
ready for guest communication. This should not happen and a bug
should be reported.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
ERROR warden: Error occurred: Guest-specific operations were attempted on a machine that is not
ready for guest communication. This should not happen and a bug
should be reported.
 INFO warden: Beginning recovery process...
 INFO warden: Calling recover: #<Vagrant::Action::Builtin::Call:0x00000002590220>
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Calling recover: VMware Middleware: Import
 INFO subprocess: Starting process: ["/usr/bin/vmrun", "list"]
 INFO subprocess: Command not in installer, restoring original environment...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
ERROR warden: Error occurred: Guest-specific operations were attempted on a machine that is not
ready for guest communication. This should not happen and a bug
should be reported.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
ERROR warden: Error occurred: Guest-specific operations were attempted on a machine that is not
ready for guest communication. This should not happen and a bug
should be reported.
 INFO warden: Beginning recovery process...
 INFO warden: Calling recover: #<Vagrant::Action::Builtin::Call:0x00000001726e50>
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO environment: Released process lock: machine-action-c1e25d94f37671eff125984c24eb1055
 INFO environment: Running hook: environment_unload
 INFO runner: Preparing hooks for middleware sequence...
 INFO runner: 2 hooks defined.
 INFO runner: Running action: environment_unload #<Vagrant::Action::Builder:0x000000020ad550>
 INFO warden: Calling IN action: HashiCorp Background Check: End
 INFO logger: Cleaning up background activation thread...
 INFO warden: Calling OUT action: HashiCorp Background Check: End
ERROR vagrant: Vagrant experienced an error! Details:
ERROR vagrant: #<Vagrant::Errors::MachineGuestNotReady: Guest-specific operations were attempted on a machine that is not
ready for guest communication. This should not happen and a bug
should be reported.>
ERROR vagrant: Guest-specific operations were attempted on a machine that is not
ready for guest communication. This should not happen and a bug
should be reported.
ERROR vagrant: /opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/machine.rb:259:in `guest'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/builtin/set_hostname.rb:21:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/home/scottm/.vagrant.d/gems/gems/vagrant-vmware-workstation-4.0.8/lib/vagrant-vmware-workstation/action_farm.rb:731:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/home/scottm/.vagrant.d/gems/gems/vagrant-vmware-workstation-4.0.8/lib/vagrant-vmware-workstation/action_farm.rb:958:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/builtin/synced_folders.rb:87:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/builtin/synced_folder_cleanup.rb:28:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/plugins/synced_folders/nfs/action_cleanup.rb:25:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/home/scottm/.vagrant.d/gems/gems/vagrant-vmware-workstation-4.0.8/lib/vagrant-vmware-workstation/action_farm.rb:1030:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/builtin/provision.rb:80:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/builder.rb:116:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/runner.rb:66:in `block in run'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/util/busy.rb:19:in `busy'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/runner.rb:66:in `run'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/builtin/call.rb:53:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/home/scottm/.vagrant.d/gems/gems/vagrant-vmware-workstation-4.0.8/lib/vagrant-vmware-workstation/action_farm.rb:76:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/home/scottm/.vagrant.d/gems/gems/vagrant-vmware-workstation-4.0.8/lib/vagrant-vmware-workstation/action_farm.rb:1060:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/builder.rb:116:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/runner.rb:66:in `block in run'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/util/busy.rb:19:in `busy'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/runner.rb:66:in `run'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/builtin/call.rb:53:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/builtin/box_check_outdated.rb:78:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/home/scottm/.vagrant.d/gems/gems/vagrant-vmware-workstation-4.0.8/lib/vagrant-vmware-workstation/action_farm.rb:1352:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/home/scottm/.vagrant.d/gems/gems/vagrant-vmware-workstation-4.0.8/lib/vagrant-vmware-workstation/action_farm.rb:278:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/home/scottm/.vagrant.d/gems/gems/vagrant-vmware-workstation-4.0.8/lib/vagrant-vmware-workstation/action_farm.rb:148:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/home/scottm/.vagrant.d/gems/gems/vagrant-vmware-workstation-4.0.8/lib/vagrant-vmware-workstation/action_farm.rb:1139:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/home/scottm/.vagrant.d/gems/gems/vagrant-vmware-workstation-4.0.8/lib/vagrant-vmware-workstation/action_farm.rb:507:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/builtin/prepare_clone.rb:15:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/builtin/handle_box.rb:56:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/builder.rb:116:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/runner.rb:66:in `block in run'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/util/busy.rb:19:in `busy'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/runner.rb:66:in `run'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/builtin/call.rb:53:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/home/scottm/.vagrant.d/gems/gems/vagrant-vmware-workstation-4.0.8/lib/vagrant-vmware-workstation/action_farm.rb:1352:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/home/scottm/.vagrant.d/gems/gems/vagrant-vmware-workstation-4.0.8/lib/vagrant-vmware-workstation/action_farm.rb:104:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/home/scottm/.vagrant.d/gems/gems/vagrant-vmware-workstation-4.0.8/lib/vagrant-vmware-workstation/action_farm.rb:278:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/builtin/config_validate.rb:25:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/home/scottm/.vagrant.d/gems/gems/vagrant-vmware-workstation-4.0.8/lib/vagrant-vmware-workstation/action_farm.rb:165:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/builder.rb:116:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/runner.rb:66:in `block in run'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/util/busy.rb:19:in `busy'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/runner.rb:66:in `run'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/machine.rb:224:in `action_raw'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/machine.rb:199:in `block in action'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/environment.rb:561:in `lock'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/machine.rb:185:in `call'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/machine.rb:185:in `action'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/batch_action.rb:82:in `block (2 levels) in run'
 INFO interface: error: Guest-specific operations were attempted on a machine that is not
ready for guest communication. This should not happen and a bug
should be reported.
Guest-specific operations were attempted on a machine that is not
ready for guest communication. This should not happen and a bug                                                     
should be reported.                                                                                                 
 INFO interface: Machine: error-exit ["Vagrant::Errors::MachineGuestNotReady", "Guest-specific operations were attempted on a machine that is not\nready for guest communication. This should not happen and a bug\nshould be reported."]

**Comment by chrisroberts** [comment]
Hi there,

Thanks for reporting this bug. I'm sure this was a real issue when originally reported (our fault for not looking sooner!) but there have been multiple Vagrant releases since the original report. I'm going to close this issue now and request you reopen the issue if you're still experiencing this problem. I'm sorry this wasn't looked at earlier :frowning_face: 

Cheers!
